### PR TITLE
Orlp 171 manage own decks backend

### PR DESCRIPTION
**Pull request by Pavelchak**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -124,4 +124,5 @@ fabric.properties
 # End of https://www.gitignore.io/api/eclipse,intellij
 /.idea/
 /target/
-/Spaced.Repetition.iml
\ No newline at end of file
+/Spaced.Repetition.iml
+/orlp.log
```
Comment (antonbakalets) [inline .gitignore]: Please also add src/main/java/com/softserve/academy/spaced/repetition/logger/logs/log.txt

File: src/main/java/com/softserve/academy/spaced/repetition/logger/logs/log.txt [hunk 2]
```diff
@@ -1 +1,7 @@
 
+2017-10-11 19:54:55,294 [WARN ][    logger.Log4jImpl] - Exception in controller = com.softserve.academy.spaced.repetition.controller.DeckController@7228b872, method = ResponseEntity com.softserve.academy.spaced.repetition.controller.DeckController.deleteDeckANDRII(Long)
```
Comment (antonbakalets) [inline src/main/java/com/softserve/academy/spaced/repetition/logger/logs/log.txt]: Please remove file from commit.

File: .gitignore (hunk 1 above)
Comment (Pavelchak) [reply .gitignore]: added to .gitignore

File: src/main/java/com/softserve/academy/spaced/repetition/logger/logs/log.txt (hunk 2 above)
Comment (Pavelchak) [reply src/main/java/com/softserve/academy/spaced/repetition/logger/logs/log.txt]: I have removed this file.

**Pull request merged**